metadata/v1.7/step_1_enable_module_event.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable module event",
+  "description": "AIP-44: The new module event framework associates every event stream with a static struct type instead of an EventHandle of current event system.",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```

Comment:
sounds code url should be linked to this repo, the source/v1.7 folder

Reply:
Thanks for finding this @sherry-x! I just fixed the link. Could you double check that? Thanks